### PR TITLE
[Term Entry] PyTorch Tensor Operations: .angle()

### LINKED ISSUE #6315: [Term Entry] PyTorch Tensor Operations: .angle()
### Reason/inspiration (optional)

We would like a new entry on the `.angle()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/angle/angle.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [x] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description
Added [Term Entry] PyTorch Tensor Operations: .angle()

### Issue Solved
Closes #6315 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/angle/angle.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+Title: '.angle()'
+Description: 'Computes the element-wise angle of complex tensors in PyTorch.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Science'
+  - 'Mathematics'
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/angle/angle.md:9]: Data science and mathematics are not in the tag list

File: content/pytorch/concepts/tensor-operations/terms/angle/angle.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+- `input`: The input complex tensor for which to compute the element-wise angle.
+- `out` (Optional): Output tensor to store the result. If provided, this tensor will be used for the output, otherwise a new tensor is created.
+
+**Return value:**
+
+A new tensor containing the element-wise angle (in radians) of the input tensor.
+
+> **Note:** Starting in PyTorch 1.8, `.angle()` returns π (pi) for negative real numbers, zero for non-negative real numbers, and propagates NaNs. In previous versions, it would return zero for all real numbers and not propagate floating-point NaNs.
+
+## Example 1: Basic usage of `.angle()`
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/angle/angle.md:38]: We can keep the title as this. 
```suggestion
## Example 1: Calculating the element-wise angle of a complex tensor
```

File: content/pytorch/concepts/tensor-operations/terms/angle/angle.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+Element-wise angles in degrees:
+  tensor([[-63.4349,   56.3099,  -45.0000],
+         [ 36.8699,  -38.6598,  161.5650],
+         [-164.0546,   14.0362,  -23.9625]])
+```
+
+This example shows how `.angle()` preserves the dimensionality of the input tensor, calculating the phase angle for each complex value while maintaining the original structure.
+
+To explore more, check out this [Intro to PyTorch and Neural Networks](https://www.codecademy.com/enrolled/courses/intro-to-py-torch-and-neural-networks) course, where you’ll learn to create, train, and test neural networks.
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/angle/angle.md:157]:
```suggestion
To learn more about how to create, train, and test neural networks, check out this [Intro to PyTorch and Neural Networks](https://www.codecademy.com/enrolled/courses/intro-to-py-torch-and-neural-networks) course on Codecademy.
```

File: content/pytorch/concepts/tensor-operations/terms/angle/angle.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+print("\nElement-wise angles in radians:\n ", radians)
+
+# Convert the angle to degrees
+degree = radians * 180 / pi
+
+# Print the computed element-wise angle in degrees
+print("\nElement-wise angles in degrees:\n ", degree)
+```
+
+This produces the following output:
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/angle/angle.md:136]:
```suggestion
This example produces the following output:
```

File: content/pytorch/concepts/tensor-operations/terms/angle/angle.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+  - 'Data Science'
+  - 'Mathematics'
+  - 'PyTorch'
+  - 'Tensor'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`.angle()`** method in PyTorch is a tensor operation that computes the element-wise angle (in radians) of a given complex tensor. This function returns the phase angle of each complex value in the tensor, representing the argument of each complex number. The `.angle()` method is particularly useful in signal processing, Fourier transforms, and other applications that involve complex number operations.
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/angle/angle.md:17]: Bcaklink to PyTorch

**Comment by mamtawardhani** [comment]
This is done @Radhika-okhade, added some more backlinks 🫡

**Review by Radhika-okhade — APPROVED** [review]

**Comment by Radhika-okhade** [comment]
Thank you for contributing to Codecademy docs @mamtawardhani. The entry looks good to be merged.
